sdk/spring-sdk/src/it/java/com/example/wiring/SpringSdkIntegrationTest.java

```diff
@@ -490,6 +490,19 @@ public void verifyForwardHeaders() throws InterruptedException {
     Assertions.assertEquals(esHeaderValue, esResponse);
   }
 
+  @Test
+  public void shouldPropagateMetadataWithHttpDeferredCall() {
+    String value = "someValue";
+
+    String actionResponse = webClient.get().uri("/action-with-meta/myKey/" + value)
```

Comment:
I wonder how I could write a similar test for java/scala sdk? Ubiquitous app?

Reply:
Are you waiting for this or do you consider the PR done? The PR looks pretty solid to me.